content/geoip/license-key-validation-api.mdx

```diff
@@ -0,0 +1,82 @@
+---
+draft: false
+title: License Key Validation API
+---
+
+MaxMind provides a License Key Validation API, a valuable tool for developers
+and organizations seeking to verify the validity of MaxMind license keys within
+their systems. This API endpoint is designed to check the existence and format
+of a license key, but it does not provide information about the license key's
+access to MaxMind products or the account to which it is associated.
+
+This API can be particularly useful when integrated with secret scanning tools
+used for compliance purposes. By using our License Key Validation API in
+conjunction, you can automate the process of finding and verifying MaxMind
+license keys in your codebase. This can help prevent unauthorized use of your
+keys and enhance the security of your applications.
+
+## Request:
+
+### Request URI
+
+API calls should be made with HTTP POST request to:
+
+`https://secret-scanning.maxmind.com/secrets/validate-license-key`
+
+### Authorization/Security
+
+There is no authorization needed for this endpoint. The request is made using
+the license key itself.
+
+### Request Body
+
+The request body may include the following field:
+
+| Key           | Value Type | Description                                                                       |
+| ------------- | ---------- | --------------------------------------------------------------------------------- |
+| `license_key` | string     | The `license_key` field should contain the license key that you want to validate. |
+
+Here is an example of how to make the request using curl:
+
+```bash
+curl -X POST https://secret-scanning.maxmind.com/secrets/validate-license-key \
```

Comment:
I wonder if it would be helpful to include `-X POST` in the curl command to avoid confusion since curl uses GET by default when `-F` isn't used. I'll remove it though.